src/coreComponents/physicsSolvers/solidMechanics/kernels/policies.hpp.in

```diff
@@ -0,0 +1,9 @@
+#ifndef GEOSX_CORECOMPONENTS_PHYSICSSOLVERSE_SOLIDMECHANICS_KERNELS_CONFIG_HPP
```

Comment:
Yeah...I was wondering if we wanted to be a little less redundant with names like this? Since it is in `solidMechanics/kernels/` do we want to include the same information in the name? i.e. `solidMechanics/kernels /SolidMechanicsPolicies.hpp.in`?

Reply:
I prefer shorter file names with more directory nesting where needed

Reply:
We have to be precise with our dependencies and cmake not to import the wrong file.

Reply:
@TotoGaz I am going to keep this the way it is, but we can revisit when we do this for the flow models. I suspect there will be some more extensive changes to be made at that time, as we will have a couple of use cases to drive design.